index.bs

```diff
@@ -2387,6 +2387,30 @@ Note: Invoking this method from a [=browsing context=] where the [=Web Authentic
 
 </div>
 
+### Deserialize Registration and Authentication ceremony options - PublicKeyCredential's `optionsFromJSON()` Method ### {#sctn-optionsFromJSON}
+
+<div link-for-hint="WebAuthentication/optionsFromJSON">
+
+[=[WRPS]=] use this method to convert JSON-compatible serialized representations of options for {{CredentialsContainer/create()|navigator.credentials.create()}} to {{PublicKeyCredentialCreationOptions}}, or JSON-compatible serialized representations of options for {{CredentialsContainer/get()|navigator.credentials.get()}} to {{PublicKeyCredentialRequestOptions}}.
+
+This method is intended to make WebAuthn API consumption tenable without requiring the use of third-party libraries.
+
+Upon invocation, the [=client=] takes the provided options and returns an identically-structured object with specific strings encoded with [=base64url encoding=] decoded to {{ArrayBuffer}} values according to the corresponding type definition.
+
+</div>
+
+### Serialize Registration and Authentication ceremony responses - PublicKeyCredential's `toJSON()` Method ### {#sctn-toJSON}
```

Comment:
This should probably be presented as an "instance method" in §5.1, like [`getClientExtensionResults()`](https://pr-preview.s3.amazonaws.com/w3c/webauthn/pull/1703.html#dom-publickeycredential-getclientextensionresults). I'm not sure if we also need to be a bit more specific about how the conversion is done; if so, the section on [converting extension inputs to CBOR](https://pr-preview.s3.amazonaws.com/w3c/webauthn/pull/1703.html#ref-for-client%E2%91%A6%E2%91%A8) could be a good example to try to follow.

Reply:
I went ahead and changed `toJSON()` to an instance method, and took a stab at defining some IDLs to help capture what should be returned.